components/password_reset_form/password_reset_form.jsx

```diff
@@ -118,11 +118,9 @@ class PasswordResetForm extends React.Component {
     }
 }
 
-PasswordResetForm.defaultProps = {
-};
 PasswordResetForm.propTypes = {
```

Comment:
We can move this to the class `static propTypes` attribute. It isn't a stopper for the PR to be merged, of course :)